Examples/microcontroller/STM32F302R6T6TR.json

```diff
@@ -0,0 +1,11407 @@
+          },
+          {
+            "siUnit": "volt",
+            "maxValue": 0.4,
+            "conditions": [
+              "TTL port",
+              "IIO = +8 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "maxValue": 1.3,
+            "conditions": [
+              "IIO = +20 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "maxValue": 0.4,
+            "conditions": [
+              "IO = +6 mA",
+              "2 V < VDD < 2.7 V"
+            ]
+          }
+        ]
+      },
+      "voh": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VDD",
+            "relativeValueModifier": -0.4,
+            "relativeValueOperator": "add",
+            "conditions": [
+              "CMOS port",
+              "IIO = +8 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "minValue": 2.4,
+            "conditions": [
+              "TTL port",
+              "IIO = +8 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VDD",
+            "relativeValueModifier": -1.3,
+            "relativeValueOperator": "add",
+            "conditions": [
+              "IIO = +20 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VDD",
+            "relativeValueModifier": -0.4,
+            "relativeValueOperator": "add",
+            "conditions": [
+              "IO = +6 mA",
+              "2 V < VDD < 2.7 V"
+            ]
+          }
+        ]
+      },
+      "absVmax": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "maxValue": 4
+          }
+        ]
+      },
+      "absVmin": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VSS",
+            "relativeValueModifier": -0.3,
+            "relativeValueOperator": "add"
+          }
+        ]
+      },
+      "vmax": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "maxValue": 3.6
+          }
+        ]
+      },
+      "imax": {
+        "values": [
+          {
+            "siUnit": "milliamp",
+            "maxValue": 25
+          }
+        ]
+      },
+      "esd": true
+    },
+    {
+      "terminalIdentifier": "34",
+      "name": "PB13",
+      "numberOfSupportedFunctions": 7,
+      "functionProperties": [
+        {
+          "perFunctionName": "PB13",
+          "direction": "bidir"
+        },
+        {
+          "perFunctionName": "TSC_G6_IO3"
+        },
+        {
+          "perFunctionName": "SPI2_SCK/I2S2_CK",
+          "interfaceType": "i2c",
```

Comment:
interface: i2s or spi. Same comments for pins with this configuration.

Reply:
yes, thanks